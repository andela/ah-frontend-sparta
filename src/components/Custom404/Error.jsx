import React from 'react';
import { Link } from 'react-router-dom';
import './ErrorStyles.css';

const Error = () => (
  <>
    <div className="heading-error">
      <h3>OOPS!! There is nothing here</h3>
    </div>
    <div className="to-dashboard">
      <Link to="/dashboard">
        <h6>Back to Dashboard</h6>
      </Link>
    </div>
  </>
);

export default Error;
