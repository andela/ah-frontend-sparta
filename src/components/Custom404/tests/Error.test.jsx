import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Error from '../Error';

describe('Error page', () => {
  it('should render without failing', () => {
    const wrapper = shallow(<MemoryRouter><Error /></MemoryRouter>);
    expect(wrapper).toMatchSnapshot();
  });
});
