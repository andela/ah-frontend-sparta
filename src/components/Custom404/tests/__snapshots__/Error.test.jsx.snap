// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page should render without failing 1`] = `ShallowWrapper {}`;
